### ISSUE TITLE
osp-core and python 3.8

### ISSUE THREAD

**Issue by paulzierep** [simphony/simphony-osp#728]
I installed osp-core (version 3.4.1) in a conda env using Python 3.8.12. 
When I run `pico list` I get the error: 

```
  File "/home/paul/anaconda3/envs/pico-test/bin/pico", line 5, in <module>
    from osp.core.pico import install_from_terminal
  File "/home/paul/anaconda3/envs/pico-test/lib/python3.8/site-packages/osp/core/__init__.py", line 3, in <module>
    import osp.core.namespaces
  File "/home/paul/anaconda3/envs/pico-test/lib/python3.8/site-packages/osp/core/namespaces.py", line 3, in <module>
    from osp.core.ontology.namespace_registry import NamespaceRegistry \
  File "/home/paul/anaconda3/envs/pico-test/lib/python3.8/site-packages/osp/core/ontology/__init__.py", line 1, in <module>
    from osp.core.ontology.oclass import OntologyClass
  File "/home/paul/anaconda3/envs/pico-test/lib/python3.8/site-packages/osp/core/ontology/oclass.py", line 2, in <module>
    from osp.core.ontology.cuba import rdflib_cuba
  File "/home/paul/anaconda3/envs/pico-test/lib/python3.8/site-packages/osp/core/ontology/cuba.py", line 42, in <module>
    rdflib_cuba = _CubaNamespace()
TypeError: __new__() missing 2 required positional arguments: 'uri' and 'terms'

```

**Comment by kysrpex** [comment]
I told you privately already how to solve this from your side. In order to solve it for everyone, I would like to ask you something.

Would `v3.4.2` also work for your use case?

**Comment by kysrpex** [comment]
@paulzierep I believe in the end you were able to use v3.8.0 right? Can the issue be closed?

**Comment by kysrpex** [comment]
I am closing the issue as this is fixed in v3.8.0.
